ansible-scylla-monitoring/defaults/main.yml

```diff
@@ -20,6 +20,7 @@ scylla_manager_ip: '127.0.0.1'
 # Download the scylla-monitoring from here
 scylla_monitoring_archive_url: 'https://github.com/scylladb/scylla-monitoring/archive/scylla-monitoring-3.10.1.tar.gz'
 
+scylla_monitoring_local_files_path: "{{ inventory_dir }}"
```

Comment:
What is "local files"? 
Why not calling it `scylla_monitoring_configuration_files_path` ?

Reply:
My idea was to clarify that this path is where the role will look for local files only and it's not related to the paths where these files will be copied to, in the remote nodes

Reply:
btw, we already have a `scylla_monitoring_config_path` variable, so adding a `scylla_monitoring_configuration_files_path` (which is exactly the remote path that the config files will be copied to) could cause some confusion

Reply:
Done